include/aos/common/config.hpp

```diff
@@ -155,6 +155,13 @@
 #define AOS_CONFIG_TYPES_PARTITION_TYPES_LEN 32
 #endif
 
+/**
+ * Max number of nodes.
+ */
+#ifndef AOS_CONFIG_TYPES_MAX_NUM_NODES
```

Comment:
Add appropriate constexpr in common/types.hpp

Reply:
Done